src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
+      <div class="stars_star star"></div>
+      <div class="stars_star star"></div>
+      <div class="stars_star star"></div>
+      <div class="stars_star star"></div>
+      <div class="stars_star star"></div>
+    </div>
+
+    <div class="stars--1">
+      <div class="stars_star star_active"></div>
+      <div class="stars_star star"></div>
+      <div class="stars_star star"></div>
+      <div class="stars_star star"></div>
+      <div class="stars_star star"></div>
+    </div>
```

Comment:
All stars elements should have the same class.
You have to manage inactive/active state in CSS using parent modifier, like `stars--1`.